shared/teams/render.js

```diff
@@ -69,13 +72,45 @@ const BetaNote = (props: BetaNoteProps) => (
   </Box>
 )
 
-type Props = {
-  onCreateTeam: () => void,
-  onJoinTeam: () => void,
-  onReadDoc: () => void,
+type TeamListProps = {
+  // TODO: Change to map to member count.
+  teams: I.Set<Teamname>,
```

Comment:
we should avoid passing immutable to the dumb components, lets convert it in mergeProps

Reply:
done